src/main/java/kg/jarkyn/cobspecserver/ListenerSocket.java

```diff
@@ -0,0 +1,31 @@
+package kg.jarkyn.cobspecserver;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+public class ListenerSocket implements Listener {
+    private ServerSocket serverSocket;
+    private boolean accepting = true;
```

Comment:
I think it's convention convention. It's just makes it a little clearer what the initial state of the object is when things are set in the constructor.